fedimint-core/src/module/audit.rs

```diff
@@ -11,20 +13,18 @@ pub struct Audit {
 
 impl Audit {
     pub fn sum(&self) -> AuditItem {
-        let mut sum = 0;
-
-        for item in &self.items {
-            sum += item.milli_sat;
-        }
+        let sum = self.items.iter().fold(0, |acc, item| acc + item.milli_sat);
```

Comment:
Yes! It requires an additional iteration since we need to map milli_sats, which is why I went with a fold. The time complexity is the same with either the fold or map / sum, and perhaps the map / sum is also a [zero-cost abstraction](https://doc.rust-lang.org/book/ch13-04-performance.html)?

Happy to change to a sum if you'd prefer.

```rust
let sum: i64 = self.items.iter().map(|item| item.milli_sat).sum();
```

Reply:
Should be zero-cost and is slightly more standard.